website/docs/concepts/html/components.md

```diff
@@ -116,7 +119,7 @@ html! {
 };
 ```
 
-When using the `with props` syntax, the children passed in the `html!` macro overwrite the ones already present in the props.
+When using the struct update syntax (passing props as `..props`), the children passed in the `html!` macro overwrite the ones already present in the props.
```

Comment:
Add the restriction that this syntax must be last in the properties list, also that fields that are set outside this syntax overwrite the base properties. 

You've used the term "struct update" here but "base props expression" in the error messages - I think we should use one term and keep it consistent with the docs and error messages. 

If you were trying to say the syntax is similar to Rust`s [Functional update syntax](https://doc.rust-lang.org/stable/reference/expressions/struct-expr.html#functional-update-syntax) then I'd be more explicit that you are making that comparison. I can read this line as _Yew is calling this syntax "struct update"_, which I don't think is correct, or quite what you meant by it.